### DISCUSSION TITLE
OneTouch and VSP Support

### DISCUSSION THREAD

**Discussion by GizmoGoody** [flz/iaqualink-py#50] [Ideas]
Is anyone actively working on OneTouch and VSP (presets and direct control) support?

Are they available at the endpoint?

**Comment by streamlineclose** [comment]
Yes - please see this suggestion I made #52

**Comment by flz** [comment]
Landed onetouch support a few weeks ago and VSP support today. Both of them could use some testing.

There's some doc on how to test dev branches [here](https://github.com/flz/iaqualink-py/wiki/Development-branch-testing). More comprehensive doc for the cli too is [here](https://flz.github.io/iaqualink-py/getting-started/cli/). Just use the master branch.

Please report feedback in discussion #202.
